src/vue/components/nav/drop-down-menu.vue

```diff
@@ -65,15 +65,15 @@
     </div>
 
     <div class="alpheios-navmenu__item" :class="{ active: $store.getters['ui/isActiveTab']('user') }"
-         v-if="Boolean(auth)" @click="changeTab('user')">
+         v-show="Boolean(auth)" @click="changeTab('user')">
```

Comment:
I didn't dive deeper - but may be the visibility condition should be the same as for navbuttons-large?

Reply:
Yes, you're right. I've missed that, will fix. Good catch!

Reply:
Will also fix a condition for wordlist.